python/cudf/cudf/tests/test_parquet.py

```diff
@@ -3407,3 +3408,27 @@ def test_parquet_reader_roundtrip_structs_with_arrow_schema():
 
     # Check results
     assert_eq(expected, got)
+
+
+@pytest.mark.parametrize("chunk_read_limit", [0, 240, 1024000000])
+@pytest.mark.parametrize("pass_read_limit", [0, 240, 1024000000])
+@pytest.mark.parametrize("use_pandas_metadata", [True, False])
+@pytest.mark.parametrize("row_groups", [[[0]], None, [[0, 1]]])
+def test_parquet_chunked_reader(
+    chunk_read_limit, pass_read_limit, use_pandas_metadata, row_groups
+):
+    df = pd.DataFrame({"a": [1, 2, 3, 4] * 1000000})
```

Comment:
I'm trying to test only large frames so that chunks are being read properly. Re-using `pdf` is probably redundant testing because we wouldn't be testing the sizes we want to test here for chunking. However, I added string type to the dataframe too. If you have any specific types in mind, I'm happy to add them here. I'm trying to keep the test combinations under control here.